### PR TITLE
Add Help and new lines to rnx-start

### PR DESCRIPTION
**Pull request by johnf**
### Description

I like to be able to hit enter in rnx-start to clear space to more easily differentiate between previous and new log messages when debugging.

For good measure I also implemented a help function that reprints the initial help message.

### Test plan

* Run react-native rnx-start
* Press "h" and see help
* Hit enter a few times and see new lines

### CODE REVIEW

File: packages/cli/src/start.ts [hunk 1]
```diff
@@ -92,6 +92,19 @@ export async function rnxStart(
...
   );
 
+  const printHelp = () => {
+    const dim = chalk.dim;
+    const press = dim(" › Press ");
+    [
+      ["r", "reload the app"],
+      ["d", "open developer menu"],
+      ["a", "show bundler address QR code"],
+      ["ctrl-c", "quit"],
```
Comment (tido64) [inline packages/cli/src/start.ts:102]: I think we should also add `h` to the help message:

```suggestion
      ["a", "show bundler address QR code"],
      ["h", "show this help message"],
      ["ctrl-c", "quit"],
```

File: packages/cli/src/start.ts [hunk 2]
```diff
@@ -251,6 +255,14 @@ export async function rnxStart(
             terminal.log(chalk.green("Reloading app..."));
             messageSocketEndpoint.broadcast("reload", undefined);
             break;
+
+          case "h":
+            printHelp();
+            break;
```
Comment (tido64) [inline packages/cli/src/start.ts:261]: Nit: Can you sort the cases?

**Review by tido64 — COMMENTED** [review]
Hi, thanks for adding this! I left a couple of comments, but it looks good otherwise.

Can you also run `yarn change` at the repo root to add a change log? The CI will fail if you don't have one.

**Comment by johnf** [comment]
@tido64 thanks, all done

**Review by tido64 — APPROVED** [review]
Thanks again, @johnf!

**Pull request merged**